common.props

```diff
@@ -37,6 +37,7 @@
     <!-- deprecated <PackageLicenseUrl>https://opcfoundation.org/license/source/1.11/index.html</PackageLicenseUrl>-->
     <PackageLicenseFile>licenses/LICENSE.txt</PackageLicenseFile>
     <PackageReleaseNotes>$(RepositoryUrl)/releases</PackageReleaseNotes>
+    <PackageReadmeFile>NugetREADME.md</PackageReadmeFile>
```

Comment:
I think the name has to be README.md, but not sure please double check

Reply:
This is not a requirement any more.